torch/_C/_dynamo/guards.pyi

```diff
@@ -91,6 +176,23 @@ class GuardManager:
         example_value,
         guard_manager_enum,
     ) -> GuardManager: ...
+    def get_root(self) -> RootGuardManager: ...
+    def get_source(self) -> str: ...
+    def fail_count(self) -> int: ...
+    def get_child_managers(self) -> list[GuardManager]: ...
+    def repr(self) -> str: ...
+    def type_of_guarded_value(self) -> str: ...
+    def get_leaf_guards(self) -> list[LeafGuard]: ...
+    def get_accessors(self) -> list[GuardManager]: ...
+    def is_guarded_value_immutable(self) -> bool: ...
```

Comment:
Should any of these is func values actually be returning TypeGuard or TypeIs?

Reply:
From my reading of the functions these do not narrow the type but are mostly used for marking the nodes for recursive dict optimizations (see https://github.com/pytorch/pytorch/pull/159183). 

cc @anijain2305 in case there are more details or I am missing context